src/com/aca/homework/week2/random/string/RandomString.java

```diff
@@ -0,0 +1,28 @@
+package com.aca.homework.week2.random.string;
+
+import java.util.Random;
+
+public class RandomString {
+    public static void main(String[] args) {
+        for (int i = 0; i <10 ; i++) {
+            returnRandomString();
+        }
+
+    }
+    public static void returnRandomString(){
```

Comment:
method name intends that it should return a String type instead of void. please change it from void to String

Reply:
this comment is still actual